### PR TITLE
Edits in Command_mp

### LINKED ISSUE #1494: Announce mob purges
When you do /mp, it doesn't announce mob purges in the chat, it only messages you with how many mobs were removed. I think we should make it so that it announces when the admins does /mp, and if this issue gets accepted, I will make the pull request. What I mean by announce is a  TFM_Util.adminAction(sender.getName() addition when the player does the command.

### PR DESCRIPTION
**Pull request by DUFCLiam**
With a small extra

Resolves #1494

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_mobspurge.java [hunk 1]
```diff
@@ -13,8 +13,8 @@
 import org.bukkit.entity.Slime;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
-@CommandParameters(description = "Purge all mobs in all worlds.", usage = "/<command>")
-public class Command_mp extends TFM_Command
+@CommandParameters(description = "Purge all mobs in all worlds.", usage = "/<command>", alias = "mp")
+public class Command_mobspurge extends TFM_Command
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_mobspurge.java]: Probably "mobpurge", but "mobspurge" could be an alias too. :)

**Comment by JeromSar** [comment]
This PR does not compile. Make sure you read the [contributing guidelines](https://github.com/TotalFreedom/TotalFreedomMod/wiki/Contributing) or you'll make me sad. :/

**Comment by DUFCLiam** [comment]
It compiled for me.... Uhm... What is happening to github?!?!

**Comment by JeromSar** [comment]
@DUFCLiam Rule of thumb: don't be hasty. I'd rather see one well thought out and thoroughly tested PR than 100 non-functional non-compiling PRs. Take your time. TFM won't go away. :)

**Comment by DUFCLiam** [comment]
Thanks :P Also @JeromSar wanna help out with CustomImmaFreedom can you review the code I have done and make sure there are no copyright infringements and if there is please make an issue telling us :) We have used some thinks from TFM_Util, The things in command (not the commands)

**Comment by DUFCLiam** [comment]
Find it here [CustomImmaFreedom](https://github.com/ImmaFreedom-Dev/CustomImmaFreedomMod)

**Pull request closed without merging**